dcrjson/dcrwalletextcmds.go

```diff
@@ -532,6 +534,30 @@ func NewWalletInfoCmd() *WalletInfoCmd {
 	return &WalletInfoCmd{}
 }
 
+type OutputDestination struct {
+	Address       string
+	Script        []byte
+	ScriptVersion uint32
+}
+
+// SweepAccountCmd defines the sweep account JSON-RPC command.
+type SweepAccountCmd struct {
+	SourceAccount         uint32
+	RequiredConfirmations uint32
+	FeePerKb              dcrutil.Amount
```

Comment:
since json-rpc request parameters are positional, and we want the required confirmations and fee to be configurable but use the wallet defaults, these two must come last.

Additionally, the json-rpc api uses json numbers between 0-21M when referring to amounts, so this must be a float64 (we should not be importing dcrutil at all here)